Find a functionally equivalent code:android.bluetooth.BluetoothSocket.flush:COMMENT
Method Modifier: 
Comment:/*package*/

@@ -1,4 +1,6 @@
 {
+    if (mSocketOS == null)
+        throw new IOException("flush is called on null OutputStream");
     if (VDBG)
         Log.d(TAG, "flush: " + mSocketOS);
     mSocketOS.flush();

